ntcore/src/main/java/edu/wpi/first/networktables/NetworkTableEntry.java

```diff
@@ -224,6 +224,26 @@ public String getString(String defaultValue) {
     return NetworkTablesJNI.getStringArray(m_handle, defaultValue);
   }
 
+  /**
+   * Checks if a data value is of a type that can be placed in a NetworkTable entry.
+   *
+   * @param data the data to check
+   * @return true if the data can be placed in an entry, false if it cannot
+   */
+  public static boolean isValidDataType(Object data) {
```

Comment:
This method should have a test added for it to make sure we hit all the branches.